mod_reforged/hooks/config/strings.nut

```diff
@@ -460,6 +460,23 @@ local vanillaDescriptions = [
  			]
 	 	})
 	},
+	{
+		ID = "perk.mastery.cleaver",
+		Key = "SpecCleaver",
+		Description = ::UPD.getDescription({
+			Fluff = "Master cleavers to inflict gruesome wounds.",
+			Requirement = "Cleaver",
+	 		Effects = [{
+ 				Type = ::UPD.EffectType.Passive,
+ 				Description = [
+					"Skills build up " + ::MSU.Text.colorRed("25%") + " less [Fatigue|Concept.Fatigue].",
```

Comment:
this is a positive effect for the character so it should be colorGreen

Reply:
Ah yeah. I just followed the vanilla style here. I think we decided to change this to green, so I'll do that.

Reply:
Actually doing it just like this will make it inconsistent with the other tooltips. I suggest we make that change in a global commit later in entire codebase once we have finalized which values we want to be green and red.